plume-front/src/main.rs

```diff
@@ -46,10 +46,15 @@ lazy_static! {
         let catalogs = include_i18n!();
         let lang = js! { return navigator.language }.into_string().unwrap();
         let lang = lang.splitn(2, '-').next().unwrap_or("en");
+
+        let english_position = catalogs
+            .iter()
+            .position(|(language_code, _)| *language_code == "en")
+            .unwrap();
```

Comment:
while i understand that this is a simple operation, i don't think we need to do it every time

Reply:
actually, never mind, this is an operation that's only dealing with things known at compile time

the optimiser will probably inline this as a constant literal